src/main/java/com/wolox/training/controllers/UserController.java

```diff
@@ -0,0 +1,75 @@
+package com.wolox.training.controllers;
+
+import com.wolox.training.dtos.BookDTO;
+import com.wolox.training.dtos.UserDTO;
+import com.wolox.training.models.Book;
+import com.wolox.training.models.User;
+import com.wolox.training.services.UserService;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+public class UserController {
```

Comment:
You could add the different status codes for the different methods,  check this 
https://www.restapitutorial.com/httpstatuscodes.html